database/database.go

```diff
@@ -316,6 +370,15 @@ func (query *SwapQuery) ToWhereClauseWithExisting(conditions []string, values []
 	if len(conditions) > 0 {
 		where = " WHERE " + strings.Join(conditions, " AND ")
 	}
+	where += " ORDER BY createdAt DESC"
```

Comment:
Wouldn't this cause issues querying tables that have no column `createdAt`?

Reply:
Yes, but its only intenteded to be used with the swap related tables right now anyways, so I think its fine for now - can be changed if needed